app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallScreen.kt

```diff
@@ -170,40 +184,71 @@ private fun OngoingCallContent(
-                    progressColor = MaterialTheme.wireColorScheme.onSurface,
-                    modifier = Modifier.align(Alignment.CenterHorizontally),
-                    size = dimensions().spacing32x
-                )
-                Text(
-                    text = stringResource(id = R.string.connectivity_status_bar_connecting),
-                    modifier = Modifier.align(Alignment.CenterHorizontally),
-                )
-            }
-        } else {
-            Box(
-                modifier = Modifier.padding(
-                    bottom = 95.dp
-                )
-            ) {
-                val topAppBarAndBottomSheetHeight = if (isCbrEnabled) APP_BAR_AND_BOTTOM_SHEET_HEIGHT_WITH_CBR_INDICATOR
-                else DEFAULT_TOP_APP_BAR_AND_BOTTOM_SHEET_HEIGHT
+        BoxWithConstraints(
+            modifier = Modifier.padding(
+                top = it.calculateTopPadding(),
+                bottom = dimensions().defaultSheetPeekHeight
+            )
+        ) {
+
+            if (participants.isEmpty()) {
+                Column(
+                    modifier = Modifier.fillMaxSize(),
+                    verticalArrangement = Arrangement.Center,
+                    horizontalAlignment = Alignment.CenterHorizontally
+                ) {
+                    WireCircularProgressIndicator(
+                        progressColor = MaterialTheme.wireColorScheme.onSurface,
+                        modifier = Modifier.align(Alignment.CenterHorizontally),
+                        size = dimensions().spacing32x
+                    )
+                    Text(
+                        text = stringResource(id = R.string.connectivity_status_bar_connecting),
+                        modifier = Modifier.align(Alignment.CenterHorizontally),
+                    )
+                }
+            } else {
+                Box(
+                    modifier = Modifier.fillMaxSize()
+                ) {
 
-                VerticalCallingPager(
-                    participants = participants,
-                    isSelfUserCameraOn = isCameraOn,
-                    isSelfUserMuted = isMuted,
-                    topAppBarAndBottomSheetHeight = topAppBarAndBottomSheetHeight,
-                    onSelfVideoPreviewCreated = setVideoPreview,
-                    onSelfClearVideoPreview = clearVideoPreview,
-                    requestVideoStreams = requestVideoStreams
-                )
+                    // if there is only one in the call, do not allow full screen
+                    if (participants.size == 1) {
+                        shouldFullScreen = false
+                    }
+
+                    // if we are on full screen, and that user left the call, then we leave the full screen
+                    if (participants.find { user -> user.id == userId } == null) {
+                        shouldFullScreen = false
+                    }
+
+                    if (shouldFullScreen) {
```

Comment:
Maybe use `Crossfade` to switch between both composables with animation? 😄

Reply:
I preferred to not use any animation, until design team decide what to use.